fastlane/Fastfile

```diff
@@ -477,13 +477,25 @@ platform :ios do
       download_dir: parent_dir_for_lprojs
     )
 
-    # Then redispatch the appropriate subset of translations back to the `InfoPlist.strings` and `Sites.strings` files in corresponding `*.lproj` dirs
-    parent_dir_for_intents_target_lprojs = File.join('WordPress', 'WordPressIntents')
+    git_commit(
+      path: File.join(parent_dir_for_lprojs, '*.lproj', 'Localizable.strings'),
+      message: 'Update app translations – `Localizable.strings`',
+      allow_nothing_to_commit: true
+    )
```

Comment:
If GitHub doesn't expand the lines on top of this diff, it might not be clear why this commit is here.

This commit is here to track the changes made by the `ios_download_strings_files_from_glotpress`. I haven't dug the history of how the action changed, but I'm guessing either that or its predecessor used to commit for us and we, rightly, removed it.